charts/backstage/templates/ingress.yaml

```diff
@@ -0,0 +1,28 @@
+{{- if .Values.ingress.enabled }}
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: {{ include "backstage.fullname" . }}
+  namespace: {{ .Release.Namespace | quote }}
+  labels: 
+    {{- include "backstage.labels" . | nindent 4 }}
+{{- if .Values.ingress.annotations }}
+  annotations:
+    {{- toYaml .Values.ingress.annotations | nindent 4 }}
+{{- end }}
+spec:
```

Comment:
do we want TLS here?

Reply:
route.openshift.io/termination: "edge"
 create TLS